lib/pages/chat/events/message.dart

```diff
@@ -680,26 +683,26 @@ class Message extends StatelessWidget {
     );
   }
 
-  // Check if the sender of the current event is the same as the previous event.
-  bool isSameSender(Event? previousEvent, Event currentEvent) {
-    // If the previous event is null, it is assumed that the message is the newest.
-    if (previousEvent == null) {
+  // Check if the sender of the current event is the same as the compared event.
+  bool isSameSender(Event? comparedEvent, Event currentEvent) {
```

Comment:
i think `previousEvent` is better name, appropriate with currentEvent name

Reply:
it shouldn't be specific to previous or next because we use this method for both previous and next